rust/src/web/graphql/feetransfers/mod.rs

```diff
@@ -253,8 +253,8 @@ impl FeetransferQueryRoot {
         // TODO
         // recipient query
         if let Some(recipient) = query.as_ref().and_then(|q| q.recipient.clone()) {
-            let mut fee_transfers: Vec<FeetransferWithMeta> = db
-                .get_internal_commands_public_key(&recipient.into())?
+            let fee_transfers: Vec<FeetransferWithMeta> = db
+                .get_internal_commands_public_key(&recipient.into(), limit as u32)?
```

Comment:
Can you limit this early? Don't you need to wait for all other filters to be complete?

Reply:
I'm going to resolve this for now. If there is more improvement that can be made, let's follow up with another PR.